### ISSUE TITLE
Is there a way to update a computedfield's last change?

### ISSUE THREAD

**Issue by simkimsia** [netzkolchose/django-computedfields#128]
let say

```python
class SomeModel(ComputedFieldsModel):
    fieldA = ...

    @computed(..., depends=[('self', ['fieldA'])])
    def comp(self):
        # do something with self.fieldA
        return ...
```

so i want a `comp_last_changed` which is a datetime

how do I do this?

**Comment by jerch** [comment]
Not directly, but you can create a dependent datetime field, that updates from changes on `comp`, something like this (untested):

```python
class SomeModel(ComputedFieldsModel):
    fieldA = ...

    @computed(..., depends=[('self', ['fieldA'])])
    def comp(self):
        # do something with self.fieldA
        return ...

    @computed(models.DateTimeField(), depends=[('self', ['comp'])])
    def comp_last_changed(self):
        return timezone.now()
```

**Comment by simkimsia** [comment]
🤔 

forgot to add that my `comp` is a boolean, so when it flips, i need to track the last_changed

or in the original `comp` method i can get the old value?

**Comment by jerch** [comment]
Right, thats not as easy as it sounds - currently the update resolver give no guarantees not to update a field in question. It resorts to "if in doubt - update anyway", since keeping values in sync is more important than anything else.

**Comment by jerch** [comment]
Thinking again about the issue the ideas above not ideal for these reasons:
- the double save approach will create havoc, if `comp_last_changed` is part of a `depends` entry of some other cf (it will turn update execution upside-down)
- an explicit kwargs on the decorator is too limited in what it can be used for (although it would work for your particular use case)

Imho a better and more versatile approach is to get signals from the update resolver done, see #58. So far I did not push that PR forward, as mass updates as normally done by the resolver create some issues about the right signal abstraction, like what types of signals we gonna need and in which transaction context the signal handlers should run. And for strict post signals (emitted after the resolver finished all transactions) it creates a lot of state to be held in memory.

Putting aside these issues for a moment, with the right signals something like this should be possible:
```python
@receiver(post_change_computedfields, sender=SomeModel, field='comp')
def my_handler(sender, field, queryset, **kwargs):
    # queryset now contains only instances, where comp changed
    # do whatever you like based on those changes
    ...
```
